W5500.device.lib.nut

```diff
@@ -526,13 +528,15 @@ class W5500.Driver {
     //      spi - configured spi, W5500 supports spi mode 0 or 3
     //      cs(optional) - configured chip select pin
     //      reset(optional) - configured reset pin
+    //      setMac(optional) - set the MAC address of the chip to the imp's MAC
     // ***************************************************************************
-    constructor(interruptPin, spi, cs = null, resetPin = null) {
+    constructor(interruptPin, spi, cs = null, resetPin = null, setMac = false) {
```

Comment:
The default behavior should not change so I believe this should default to true.

Reply:
Oops, missed that one. Fixed in https://github.com/electricimp/Wiznet_5500/pull/10/commits/f6c8b006e40f1e763982f009f32f21c294854a5c